Quick_Hull/Quick_hull.cpp

```diff
@@ -0,0 +1,132 @@
+/*
+Quickhull Algorithm for Convex Hull
+
+Given a set of points, a Convex hull is the smallest convex polygon containing all the given points.
+
+Input is an array of points specified by their x and y coordinates. Output is a convex hull of this set of points in ascending order of x coordinates.
+
+*/
+
+
+#include<iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
+#define ptspr pair<int,int>
+
+set<ptspr> ipts;
+set<ptspr> finalpts;
+
+
+int findside(ptspr p1,ptspr p2,ptspr p)
+{
+    int val=(p2.first-p1.first)*(p.second-p1.second)-(p2.second-p1.second)*(p.first-p1.first);
```

Comment:
- Break this line. Should not contain more than 80 characters.
- Add space around operators.